File: pset7/mario/more/mario.py
from cs50 import get_int

x = 0
while x < 1 or x > 8:
    x = get_int("Height: ")

for i in range(x):

    #Makes beginning spaces
    for j in range(x-(i+1)):
        print(" ", end="")

    #Creates first pyramid
    for j in range(i+1):
        print("#", end="")

    #Makes the space in the middle
    print(" ", end="")

    #Creates the second pyramid
    for j in range(i+1):
        print("#", end="")

    print("")

File: pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n, j;

    do
    {
        n = get_int("Height: ");
    }
    while (n < 0 || n > 23);

    if (n == 0)
    {
        printf("\n");
    }
    else
        for (int i = 1; i <= n; i++)
        {
            printf("%*s", n - i, ""); //creates first set of spaces

            for (j = 0; j < i; j++)
            {
                printf("#"); //creates first pyramid
            }

            printf(" ");

            for (j = 0; j < i; j++)
            {
                printf("#");    //creates second pyramid
            }
            printf("\n"); //makes new line
        }
}